new-feature.txt

```diff
@@ -0,0 +1,2 @@
+this file was created in the freature1 branch
```

Comment:
its's really good that you have decided to add this line of code

Reply:
LGM

Reply:
approved